packages/modal/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@leafygreen-ui/modal",
+  "version": "0.9.0",
+  "description": "LeafyGreen UI Kit Modal",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js", 
+    "ts:emit": "tsc --project ./tsconfig.json"
+  },
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@leafygreen-ui/lib": "^3.0.0",
+    "@leafygreen-ui/portal": "^1.1.0",
+    "@leafygreen-ui/palette": "^1.0.0",
+    "@leafygreen-ui/hooks": "^1.0.0",
+    "@leafygreen-ui/icon": "^1.0.4",
+    "lodash": "^4.17.11"
+  },
+  "gitHead": "53385cfed5ecf3101d098d26dedc6a40b27c402a"
```

Comment:
This can be removed until lerna adds it.

Reply:
👍